modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -129,7 +129,8 @@ $ echo "    bootstrapOSImage=${BOOTSTRAP_OS_IMAGE}"
 ----
 platform:
   baremetal:
-    bootstrapOSImage: http://<BAREMETAL_IP>:8080/<RHCOS_QEMU_PATH>?sha256=<RHCOS_QEMU_UNCOMPRESSED_SHA256>
+    bootstrapOSImage: <bootstrap_os_image>
```

Comment:
You'll want to ad <1> at the end of this line (otherwise the callout won't render properly). So:

```
    bootstrapOSImage: <bootstrap_os_image> <1>
```